tests/Midi_gtest.cpp

```diff
@@ -0,0 +1,46 @@
+#include <gtest/gtest.h>
+#include "hid/midi.h"
```

Comment:
You can include "system.h" to fix the build error. The `System::Now()` function is already available in a unit test compatible, mocked form